docs/tutorial/yurt-app-manager.md

```diff
@@ -0,0 +1,253 @@
+
+# Use yurt-app-manager to manage your edge node and workload
+
+In this tutorial, we will show how the yurt-app-manager helps users manage 
+there edge nodes and workload.
+Suppose you have a Kubernetes cluster in an Openyurt environment, or a native Kubernetes cluster with at least two nodes.
+
+## Install yurt-app-manager
+
+### 1. create and push yurt-app-manager image
```

Comment:
I suggest to skip the details of step 1 & 2. We could prepare ready images in a public repo and checkin the test yamls in the tutorial directory.

Reply:
done, I added an new file `yurt-app-manager-dev.md` just for developers.